run_tests.py

```diff
@@ -135,28 +145,6 @@
 ##############################################################################
 
 print('testing extra observations')
-class EmptyEnvWithExtraObs(gym_minigrid.envs.EmptyEnv5x5):
```

Comment:
This appears to be a purely testing environment and not an environment intended to be used.
Is there a reason why you have removed it from here

Reply:
I did not find a way to register it in `run_tests.py` (so that it can be created with `gym.make` - which, IIRC, was necessary to use the wrapper. I agree that it's a purely testing environment. Do you have any idea of how it can be registered and use in the same file ?

Reply:
Why does the environment need to use `gym.make`? Currently the testing doesn't use it like that

Reply:
Ah ! Makes sense. For some reason, I thought `gym.make` was necessary to use one of the 4 wrappers being tested. I will revert this change. Thank you